src/components/organisms/SfBottomNavigation/SfBottomNavigation.stories.js

```diff
@@ -0,0 +1,19 @@
+import { storiesOf } from "@storybook/vue";
+import { withKnobs, text } from "@storybook/addon-knobs";
+import SfBottomNavigation from "./SfBottomNavigation.vue";
+
+export default storiesOf("BottomNavigation", module)
+  .addDecorator(withKnobs)
```

Comment:
adjust stories to common format (check banner) 
- props
- scss mods
- one for every prop

also include  markdown notes (search for `notes` in other stories)